controllers/restic.go

```diff
@@ -36,9 +36,17 @@ func (r *VolumeSnapshotBackupReconciler) CreateVSBResticSecret(log logr.Logger)
 		r.Log.Error(err, "unable to fetch PVC")
 		return false, err
 	}
+
+	// get restic secret name
+	rs := resticSecretName
```

Comment:
So by if the user does not specify a secret name we assume `restic-secret` as the name ?

Reply:
Just saw the operator PR, we will be creating `restic-secret` for the user, right ?

Reply:
yes. Let me  ensure we are checking that in oadp PR. I dont think I am right now. Good catch

Reply:
we create `<bslname>-restic-secret` for every bsl on behalf of the user. `restic-secret` is a catch all.

Reply:
Just a suggestion, might be worth naming this something like `volsync-restic-secret` just something to indicate that it's separate from the main velero restic usage

Reply:
@dymurray good point. I was quite confused while switching back and forth. Would it help if I change `restic-secret` -> `volsync-restic-secret` in both places, OADP controller & vol snap controller?

Reply:
updated `restic-secret` -> `dm-restic-secret`